### DISCUSSION TITLE
post/isolated-functions-servicebus-message-metadata

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#9] [Announcements]
# post/isolated-functions-servicebus-message-metadata

Mark Heath's Development Blog

https://www.markheath.net/post/isolated-functions-servicebus-message-metadata

**Comment by SeanFeldman** [comment]
I hope "Sean's property" will be replaced by a proper object some day 😀
Fingers crossed soon enough.

**Reply by viorelbuligadev** [reply]
Nice article! I used this approach a few weeks ago, I added a correlationId in ApplicationProperties.
